src/api/apiPlanClass.tsx

```diff
@@ -1,9 +1,5 @@
 import api from './api';
 
-export const getPlanClasses = (id: string | undefined) => {
```

Comment:
since the folder name is already `api`, maybe the file name should just be `planClasses`?